jekyll/_data/sidenav.yml

```diff
@@ -418,6 +418,9 @@ ja:
         - name: Supported Languages
           i18n_name: サポートされている言語
           link: ja/2.0/demo-apps/
```

Comment:
Maybe you want to clean up this one since this is from another PR.

Reply:
sidenav.yml shouldn't be here.  I'll close this PR and create a new one.